verdigado_template_default/__manifest__.py

```diff
@@ -14,7 +14,7 @@
         "base",
         "account",
         "l10n_de",
-        "account_sepa_direct_debit",
+        "account_banking_sepa_direct_debit",
```

Comment:
rate ich hier richtig?

Reply:
Nee.  Das Modul heißt wirklich `account_sepa_direct_debit`

Reply:
in welchen repo lebt das?